lib/barcelona/plugins/pcidss_plugin.rb

```diff
@@ -238,6 +413,180 @@ def build_resources
             tag("barcelona-role", "pcidss")
           ]
         end
+
+        add_resource("AWS::EC2::Volume", "OSSECManagerVolume") do |j|
+          j.AvailabilityZone "ap-northeast-1a"
```

Comment:
Why do you use `district.stack_resources["SubnetTrusted1"]` here, as "OSSECManagerASG"?